test.js

```diff
@@ -1 +1,3 @@
 console.log('In test.js'); 
+
+console.log('Hello World!'); 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://54.144.41.16/coding_rules#rule_key=javascript%3AS2228)